ui/seek_bar_factory.js

```diff
@@ -0,0 +1,26 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.ui.SeekBarFactory');
+
+goog.require('shaka.ui.SeekBar');
+
+/**
+ * @implements {shaka.extern.ISeekBarFactory}
+ * @export
+ */
+
+shaka.ui.SeekBarFactory = class {
```

Comment:
In keeping with convention, please call this `shaka.ui.SeekBar.Factory` and move it to the same file as `shaka.ui.SeekBar`.  Thanks!

Reply:
Done